V3 validation

sadhasivam-ladybirdweb opened the pull request:
## Fixes  

- #3435 
- #3389 
- #3514

## Commands to Run  
```bash
php artisan db:seed --class="Database\Seeders\v4_0_2_4_RC_1\DatabaseSeeder"

```

In this PR, I added extra validation in V3 and also added a CSP header. This CSP check has been included on all pages that load external resources, so we can compare it with the development environment to check it break anything. Additionally, the dashboard load time has been reduced.

sandesh556 reviewed (commented):
Is honey pot added? Also we need to write some mock tests for this feature

sadhasivam-ladybirdweb commented:
> Is honey pot added? Also we need to write some mock tests for this feature

Added honeypot to the login, register, reset password, and forgot password forms; also applied it to the demo and contact forms. Enhanced the honeypot implementation and added test cases for CSP and login honeypot.

AbishekChinnaraju commented:
@sadhasivam-ladybirdweb Please resolve the conflicts

AbishekChinnaraju commented:
Round One Testing Summary:
Issue sheet: https://docs.google.com/spreadsheets/d/1RnrKltCX8cwVNVHDZzq4UPc5kWlPUhrTL8FpyouHTmQ/edit?gid=0#gid=0
**Note**
- I mentioned a few issues on the above sheet which are missed in development related to reCAPTCHA. Please fix it here.

indira15v commented:
**Round two testing summary**
Please fix the updated issues

sadhasivam-ladybirdweb commented:
https://github.com/xHossein/PyPasser
@AbishekChinnaraju  @indira15v 

This is a tool we can use to bypass reCAPTCHA. I’ve tested a v3 in the development branch, and it successfully bypassed reCAPTCHA v3. However, it doesn’t seem to work in the current branch. See if you can test it with reCAPTCHA v2 as well. Also, check if you can find any similar packages that can bypass v2 or v3.

AbishekChinnaraju commented:
@sadhasivam-ladybirdweb Please resolve the conflicts.

indira15v commented:
**Round two testing summary**
Tested this PR on my  local existing instance
The mentioned issues  are fixed
Bypassed  v3 reCAPTCHA with the help of the developer and wrote Python script
![image](https://github.com/user-attachments/assets/93e11460-af05-41be-8f03-bb8b960d5e65)
Checked dashboard loading time
